influxdb/content.md

```diff
@@ -1,643 +1,221 @@
-# What is InfluxDB?
+**This README covers all currently supported versions of InfluxDB:**
 
-InfluxDB is the time series data platform designed to handle high write and query workloads. Using InfluxDB, you can collect, store, and process large amounts of timestamped data, including metrics and events for use cases such as DevOps monitoring, application metrics, IoT sensors, and event monitoring.
+-	**InfluxDB 3** (Core and Enterprise)
+-	**InfluxDB v2**
+-	**InfluxDB v1**
 
-Use the InfluxDB Docker Hub image to write, query, and process time series data in [InfluxDB v2](https://docs.influxdata.com/influxdb/v2/) or [InfluxDB v1](https://docs.influxdata.com/influxdb/v1/).
+Scroll to the appropriate section below for Docker setup instructions, configuration flags, and usage examples.
 
-For more information, visit https://influxdata.com.
+# InfluxDB Docker Image Overview
 
 %%LOGO%%
 
-# How to use this image for InfluxDB v2
+InfluxDB is the time series database platform designed to collect, store, and process large amounts of timestamped data. InfluxDB supports high write and query workloads and is commonly used for:
 
-**Quick start**: See the guide to [Install InfluxDB v2 for Docker](https://docs.influxdata.com/influxdb/v2/install/?t=Docker) and get started using InfluxDB v2.
+-	Monitoring infrastructure and applications
+-	Collecting data from IoT devices
+-	Storing log and event data
 
-## Start InfluxDB v2 and set up with the UI, CLI, or API
+InfluxDB offers multiple versions and deployment options to meet diverse technical and operational requirements.
 
-To start an InfluxDB v2 container, enter the following command:
+## InfluxDB 3
 
-```bash
-docker run \
-    -p 8086:8086 \
-    -v "$PWD/data:/var/lib/influxdb2" \
-    -v "$PWD/config:/etc/influxdb2" \
-    %%IMAGE%%:2
-```
-
-Replace the following with your own values:
+InfluxDB 3 is a new InfluxDB engine that uses an object store-backed architecture. It stores time series and event data using Apache Arrow and Parquet. It supports high-ingest workloads and fast queries.
```

Comment:
```suggestion
InfluxDB 3, the latest InfluxDB engine, uses Apache Arrow in-memory processing, the DataFusion SQL engine, and diskless
  object storage architecture. It delivers sub-10ms query response times, unlimited cardinality, and supports both SQL and InfluxQL.
```

- Don't include Parquet
- Callout the query engine and SQL and InfluxQL support.